db/migrate/20250217003916_x_summary_large_image_based_on_deprecated_setting.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class XSummaryLargeImageBasedOnDeprecatedSetting < ActiveRecord::Migration[7.2]
+  def up
+    old_setting =
+      DB.query_single(
+        "SELECT value FROM site_settings WHERE name = 'twitter_summary_large_image' LIMIT 1",
+      ).first
+
+    if old_setting.present?
+      DB.exec(
+        "INSERT INTO site_settings(name, value, data_type, created_at, updated_at)
+        VALUES('x_summary_large_image', :setting, '18', NOW(), NOW())
+        ON CONFLICT DO NOTHING",
```

Comment:
I don't think we need `ON CONFLICT`? It should never happen unless we somehow used this setting name previously?